sunpy/sun/models.py

```diff
@@ -9,7 +9,14 @@
   as defined in Table 6 of :cite:t:`turck-chieze_revisiting_1988`.
 * :func:`~sunpy.sun.models.differential_rotation`: Function for calculating
   solar differential rotation for different models
+
+Additionally, this module provides functions to load 1D solar models,
+which are:
+
+* :cite:t: `avrett_loeser_2008` called "avrett_loeser_2008"
```

Comment:
This new model should be listed right next two the first models.  The fact that it dynamically loads a model from a file is merely an implementation detail, and not relevant for the docstring.